qiskit_ibm_runtime/accounts/account.py

```diff
@@ -48,17 +49,30 @@ def _assert_valid_url(url: str) -> None:
     try:
         urlparse(url)
     except:
-        raise ValueError(f"Inappropriate `url` value. Failed to parse '{url}' as URL.")
+        raise ValueError(f"Invalid `url` value. Failed to parse '{url}' as URL.")
+
+
+def _assert_valid_proxies(config: ProxyConfiguration) -> None:
```

Comment:
Why did you decide to use private functions in `account.py` instead of making them part of the `Account` class?